src/scheduler/buckets.cpp

```diff
@@ -1077,7 +1077,7 @@ check_node_buckets(status *policy, server_info *sinfo, queue_info *qinfo, resour
 	 * If it doesn't exist, we'll create it and add it to the cache
 	 */
 	if (resresv->place_spec->group != NULL) {
-		char *grouparr[2] = {0};
+		string_vector groupvec{resresv->place_spec->group};
```

Comment:
While the counts_map typedef is converting a long type into a shorter type, std::vector<std::string> isn't very long.  Use the explicit type.